.github/workflows/sync-fork.yaml

```
@@ -0,0 +1,39 @@
+          git fetch upstream
+          git checkout ${{ github.ref_name }} || git checkout -b ${{ github.ref_name }}
+
+      - name: Force Merge Upstream Changes
+        run: |
+          # Dynamically merge the corresponding upstream branch
+          git merge upstream/${{ github.ref_name }} --allow-unrelated-histories --strategy-option=theirs --no-edit
+
+      - name: Push Changes to Fork
+        run: |
+          echo "Pushing changes to forked repository..."
+          git push origin ${{ github.ref_name }} --force
```

yamllint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)